graphics/osl/Portfile

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               qt5 1.0
+PortGroup               cmake 1.1
+
+github.setup            imageworks OpenShadingLanguage 1.10.9 Release-
+name                    osl
+categories              graphics
+platforms               darwin
+license                 BSD
+maintainers             @jasonliu-- openmaintainer
+homepage                http://opensource.imageworks.com
+description             advanced shading language for computer \
+                        graphics rendering
+long_description        Open Shading Language is a language developed \
+                        by Sony Pictures Imageworks, used for \
+                        programming shader effects in \
+                        computer-generated imagery. It can be used to \
+                        define how surfaces or volumes scatter light, \
+                        and can be used for describing materials, \
+                        lights, displacement, and pattern generation.
+
+checksums               rmd160  e03ba154633bdd2860e458e39818be2af00d1ee5 \
+                        sha256  6a62e89797c0dac382cb1816f87ae53deb332a4aedb89c956f05b36cd261a9a4 \
+                        size    13547442
+
+# Keep this value synchronized with the
+# newest LLVM that is compatible with OSL
+set llvm_version 9.0
+
+depends_build-append    port:flex \
+                        port:bison
+depends_lib             port:llvm-$llvm_version \
+                        port:boost \
+                        port:ilmbase \
+                        port:pugixml \
+                        port:openimageio
+qt5.depends_component   qtbase
+
+post-patch {
+    reinplace "s|llvm-config|llvm-config-mp|g" \
```

Comment:
This replacement is incorrect. The macports `llvm-config` command to use is versioned. e.g. it should be `llvm-config-mp-${llvm_version}`

Reply:
I know it doesn't look right from inside the portfile, but inside file I'm patching, the string actually starts off as `llvm-config-${LLVM_VERSION}`. So my replacement really does result in `llvm-config-mp-${LLVM_VERSION}`. :)

Reply:
ah right, I see.